app/lib/actions/katello/capsule_content/sync.rb

```diff
@@ -27,17 +27,31 @@ def plan(capsule_content, options = {})
               need_updates.each do |repo|
                 plan_action(Pulp::Repository::Refresh, repo, capsule_id: capsule_content.capsule.id)
               end
-
               plan_action(ConfigureCapsule, capsule_content, environment, content_view)
+              sync_repos_to_capsule(capsule_content, repository_ids)
+              plan_action(RemoveOrphans, :capsule_id => capsule_content.capsule.id)
```

Comment:
that weekly cron job don't exist on the capsule.  The way it works on the katello server is to use a rake task to kick it off, but since we can't do that on the capsule we'd need some other source of auth.  Easier to just do it at the end of a sync.